### PR TITLE
Give love to index creation

### PR DESCRIPTION
**Pull request by romainneutron**

### CODE REVIEW

File: resources/scripts/indexing/createIndex.sh [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
-                            "lowercase", "standard", "asciifolding"
+curl -s -X POST $1 -d '
+{
+    "settings":{
+        "index":{
+            "analysis":{
+                "analyzer":{
+                    "city":{
+                        "type":"custom",
+                        "tokenizer":"standard",
```
Comment (romainneutron) [inline resources/scripts/indexing/createIndex.sh]: should be whitespace

**Comment by romainneutron** [comment]
This PR fixes #44

**Comment by romainneutron** [comment]
@nlegoff PR ready for final review

File: README.md [hunk 2]
```diff
@@ -255,171 +248,199 @@ Returns the city in which the given IP address is located
...
-on GitHub, we will be glad to merge your code.
-
-## Testing
-
-This server relies on `Mocha <http://visionmedia.github.com/mocha/>`_
-and `Supertest <https://github.com/visionmedia/supertest>`_ for unit testing.
-All you have to do is to run the following command in the root folder:
-
 ```
 make test
```
Comment (nlegoff) [inline README.md:459]: you should remove this line as well no ?

Comment (romainneutron) [inline README.md:459]: no, I should revert the two previous blocks

File: resources/scripts/indexing/cities [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+                  ]
+               }
+            }
+         }
+      }
+   }
+}
+'
+
+curl -XPOST "http://localhost:9200/geonames2/cities/_mapping" -d '
```
Comment (nlegoff) [inline resources/scripts/indexing/cities]: curl -XPOST "http://localhost:9200/geonames/cities/_mapping" -d '

File: routes/city.js [hunk 4]
```diff
@@ -72,12 +76,11 @@ module.exports = function(app) {
...
                     point,
                     app.get('req.sort'),
-                    app.get('req.limit'),
-                    app.get('req.ord')
+                    app.get('req.limit')
             );
 
             request({
-                uri: app.get('es.connection.string')('cities'),
+                uri: app.get('es.connection.string')('cities')+'?pretty=true',
```
Comment (nlegoff) [inline routes/city.js]: is it necessary to prettify ?

**Comment by romainneutron** [comment]
comments addressed

**Pull request merged**